include/openmc/distribution.h

```diff
@@ -8,7 +8,6 @@
 
 #include "pugixml.hpp"
 #include <gsl/gsl-lite.hpp>
-
```

Comment:
This blank line is probably intentional - it's a style practice to separate headers that are from the system and/or other tools from headers that are from this project.